tictactoe/script.js

```diff
@@ -20,69 +18,54 @@ const WinCombinations = [
     [6, 4, 2]
 ];
 
-//ZMIANA NA FUNCKJE STRZALKOWA
-function checkForWin(){
+const checkForWin = () =>{
 
-    // for(let i = 0; i <=7; i++){
-    //     const [number1, number2, number3] = WinCombinations[i];
-    //     if(fields[number1] !== "" && fields[number1] === fields[number2] && fields[number1] === fields[number3]){
-    //         alert(`Player ${activePlayer} won!`);
-    //         return true;
-    //     }
-    // }
-    WinCombinations.forEach((combination)=>{
+    WinCombinations.forEach((combination)=>{ 
         const [number1, number2, number3] = combination;
-        //SPRAWDZENIE CZY POLE JEST PUSTE
-     if(fields[number1] === fields[number2] && fields[number1] === fields[number3]){
-        won = true; 
-            alert(`Player ${activePlayer} won!`);
-
-            //CO DAJE RETURN (DEBBUGER)
-            return true;
-
-        }
+        if(fields[number1] !== "" && fields[number1] === fields[number2] && fields[number1] === fields[number3]){
+                won = true; 
+                alert(`Player ${activePlayer} won!`);
+                gameActive = false;
+            }
     })
 
-
     if(moves === 9 && !won){
             alert("Its draw!");
-            result.innerHTML="Its draw";
-            //CO DAJE RETURN (DEBBUGER)
-            return true;
-        }
+            gameActive = false;
+    }
 };
 
+const changePlayer = () =>{
+    activePlayer = activePlayer === "X" ? "O" : "X";
+    playerName.innerHTML=`Player ${activePlayer} move`;
+};
 
-// Event listner on game board
-document.querySelector("#gameBoard").addEventListener("click", function(e) {
-	// Check if clicked button match class 'field'
-
-    debugger;
-
-	if(e.target?.matches(".field")) {
-		
-		const {number} = e.target.dataset;
-        //Check if choosed field is empty
-        if(fields[number] === ""){
-            //Add player chose to field [X/O]
-            moves++
-            fields[number] = activePlayer;
-            //Add player chose to button text
-            e.target.innerHTML= activePlayer;
-            //Check for win function
-            checkForWin();
-            //Do this if there is a win or draw
-            if(checkForWin()){
-                if (confirm("Press okay to play again")) {
-                    window.location.reload();
-                    }
+const isGameActive = () =>{
+    if(!gameActive){
+        if (window.confirm("Do you want to play again?")) {
+            gameActive = true;
+            window.location.reload();
+          }else{
+            gameActive = false;
+          }
+    }
+}
+    document.querySelector("#gameBoard").addEventListener("click", function(e) {
+        if(gameActive){
+            if(e.target?.matches(".field")) {

```

Comment:
white characters are not in the same pattern, each indent should be same as the other. Opening "{" bracket should be everywhere in the same pattern with space before.

look at prettier doc